websites/I/India Today News/presence.ts

```diff
@@ -0,0 +1,541 @@
+	} else if (pathname.includes("/photo")) {
+		presenceData.details = document.querySelector(
+			"h1.Photo_maintitle__G_cdj"
+		)?.textContent;
+		presenceData.state = `Total of ${
+			document.querySelector("span.typedata")?.textContent.split("/")[1]
+		} photos`;
+		presenceData.largeImageKey = document
+			.querySelector("div.PhotoCard_card__pic__axT6x")
+			?.querySelector("img")?.src;
+		presenceData.smallImageKey = Assets.Logo;
+		presenceData.smallImageText = "India Today News - Photos";
+		presenceData.buttons = [
+			{
+				label: "View article",
+				url: href,
+			},
+		];
+
+		if (pathname.includes("s") && !pathname.includes("/photo/")) {
+			presenceData.details = `Viewing ${pathname
+				.split("/photos")[0]
+				?.replace("/", "")
+				?.split("/")[1]
+				.toUpperCase()} Photo Gallery`;
+			presenceData.largeImageKey = Assets.Logo;
+			presenceData.smallImageKey = Assets.Reading;
+			presenceData.smallImageText = "India Today News - Photo Gallery";
+			presenceData.buttons = [
+				{
+					label: "View Photo Gallery",
+					url: href,
+				},
+			];
+			delete presenceData.state;
+		}
+	} else if (video && !isNaN(video.duration) && pathname.includes("/video")) {
+		const [startTimestamp, endTimestamp] = presence.getTimestamps(
+			Math.floor(video.currentTime),
+			Math.floor(video.duration)
+		);
+		presenceData.details = document.querySelector(
+			"h1.Videoplayer_title__3Vk_I"
+		)?.textContent;
+		presenceData.state = storyDate;
+		presenceData.smallImageKey = video.paused ? Assets.Pause : Assets.Play;
+		presenceData.smallImageText = video.paused ? strings.pause : strings.play;
+		presenceData.buttons = [
+			{
+				label: "Watch video",
+				url: href,
+			},
+		];
+
+		if (showTimestamp) {
+			presenceData.startTimestamp = startTimestamp;
+			presenceData.endTimestamp = endTimestamp;
+		}
+
+		if (video.paused) delete presenceData.endTimestamp;
+
+		if (pathname.includes("s") && !pathname.includes("/video/")) {
+			presenceData.details = `Viewing ${pathname
+				.split("/videos")[0]
+				?.replace("/", "")
+				?.split("/")[1]
+				.toUpperCase()} Videos`;
+			presenceData.smallImageKey = Assets.Reading;
+			presenceData.smallImageText = "India Today News - Videos";
+			presenceData.buttons = [
+				{
+					label: "View Videos",
+					url: href,
+				},
+			];
+		}
+	} else if (pathname.includes("/weather/")) {
+		presenceData.details = document
+			.querySelector("div.wtr_ttl_lhs")
+			?.querySelector("h1")
+			?.textContent.split("Today")[0];
+		presenceData.state = document
+			.querySelector("div.wtr_ttl_lhs")
+			?.querySelector("h1")
+			?.textContent.split("Today")[1];
+		presenceData.smallImageKey = document
+			.querySelector("span.wkl_li_icn")
+			?.querySelector("img")?.src;
+		presenceData.smallImageText = `${
+			document.querySelector("span.wtr_tmp_txt")?.textContent
+		} - ${
+			document.querySelector("div.wtr_tmp_rhs")?.querySelector("strong")
+				?.textContent
+		}°C`;
```

Comment:
Let's also trim here cuz lots of unnecessary spaces.
```suggestion
		presenceData.smallImageText = `${
			document.querySelector("span.wtr_tmp_txt")?.textContent.trim()
		} - ${
			document.querySelector("div.wtr_tmp_rhs")?.querySelector("strong")
				?.textContent.trim()
		}°C`